src/app/series/directives/series-feeds.component.ts

```diff
@@ -12,6 +12,49 @@ export class SeriesFeedsComponent implements OnDestroy {
   podcast: FeederPodcastModel;
   loading = true;
   expanded = [];
+  includeTagOptions: string[][];
+  episodeOffsetOptions = [
+    ['5 minutes', 300],
+    ['15 minutes', 900],
+    ['1 hour', 3600],
+    ['6 hours', 21600],
+    ['12 hours', 43200],
+    ['24 hours', 86400]
```

Comment:
I just randomly picked some values here.  I'm actually not sure windowing will be useful to anyone yet?  Especially not until we support negative values - it's a bit confusing to set a positive offset on the default/public feed, and leave it blank on the private/subscriber feed.